src/UiPatternInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Drupal\ui_patterns;
+
+/**
+ * Interface UiPatternInterface.
+ *
+ * @package Drupal\ui_patterns
+ */
+interface UiPatternInterface {
```

Comment:
This is the interface of the Pattern plugin. It needs to stay there. Plugins must inherit this class to be found by the UiPatternManager's discovery system.